### ISSUE TITLE
Talaria drain job should provide an ability to drop specific devices.

### ISSUE THREAD

**Issue by Sachin4403** [xmidt-org/talaria#312]
As a part of this issue, we need to enhance the Talaria Drain job so that it will be able to drop specific devices based on the input. We can extend this via these two ways either we can expect device Id or based on regex we can drop specific set of devices.

cc: @schmidtw

**Comment by s-srakshe** [comment]
Can we have this based also on partner Id ? I mean like gating can we allow to drain devices matching specific partner id[s]

**Comment by Sachin4403** [comment]
@s-srakshe  yes we can have both filters i.e. ParnerId and Optional Mac address or Mac address regex based on this talaria will drop the devices.
